widget/view.html

```diff
@@ -3,166 +3,272 @@
+                                                    show="(initList.length > 0)">
+                                                    <div ng-bind="option"></div>
+                                                </ui-select-choices>
+                                            </ui-select>
+                                            <span class="font-075em padding-left-10 text-danger"
+                                                data-ng-if="invalidIPs.length > 0">{{viewWidgetVars.EXCLUDELIST_CONFIG_PAGE_IP_ADDRESS_ERROR_MESSAGE}}<strong>{{invalidIPs.join(',')}}</strong>
+                                            </span>
+                                        </div>
+                                    </div>
+                                </div>
+                                <div>
+                                    <div class="padding-bottom-md">
+                                        <label
+                                            class="control-label text-transform-none">{{viewWidgetVars.EXCLUDELIST_CONFIG_PAGE_URL_LABEL}}:
+                                            <span data-uib-tooltip="{{viewWidgetVars.EXCLUDELIST_CONFIG_PAGE_URL_TOOLTIP}}"
+                                                data-tooltip-placement="top-left" data-tooltip-append-to-body="true"><i
+                                                    class="icon icon-information" aria-hidden="true"></i></span>
+                                        </label>
+                                        <div class="cs-select padding-left-3">
+                                            <ui-select name="excludedURLList"
+                                                data-ng-model="updatedGlobalSettings['sfsp-excludelist-urls'].recordValue"
+                                                multiple class="custom-multi-select" tagging="undefined"
+                                                tagging-label="false"
+                                                data-ng-change="validateIOC($select.selected, 'sfsp-excludelist-urls')">
+                                                <ui-select-match
+                                                    placeholder="{{viewWidgetVars.EXCLUDELIST_CONFIG_PAGE_URL_PLACEHOLDER}}">{{$item}}</ui-select-match>
+                                                <ui-select-choices repeat="option in initList" refresh-delay="0"
+                                                    show="(initList.length > 0)">
+                                                    <div ng-bind="option"></div>
+                                                </ui-select-choices>
+                                            </ui-select>
+                                            <span class="font-075em padding-left-10 text-danger"
+                            data-ng-if="invalidURLs.length > 0">{{viewWidgetVars.EXCLUDELIST_CONFIG_PAGE_URL_ERROR_MESSAGE}}<strong>{{invalidURLs.join(', ')}}</strong></span>
```

Comment:
Better to create invalidURLs.join(', ') this functionality in controller js

Reply:
DONE